src/platform/packages/shared/kbn-scout/src/playwright/page_objects/date_picker.ts

```diff
@@ -10,9 +10,19 @@
 import type { ScoutPage } from '..';
 import { expect } from '..';
 
+export enum DateUnitSelector {
+  Seconds = 's',
+  Minutes = 'm',
+  Hours = 'h',
+}
+
 export class DatePicker {
   constructor(private readonly page: ScoutPage) {}
 
+  public readonly defaultStartTime = 'Sep 19, 2015 @ 06:31:44.000';
+  public readonly defaultEndTime = 'Sep 23, 2015 @ 18:31:44.000';
+  public readonly endTimeNoResults = 'Sep 19, 2015 @ 18:45:00.000';
```

Comment:
let's avoid it. Each plugin/package consuming scout should define its own defaults. It should prevent copy/paste and make tests more predictable.